packages/preset-mini/src/rules/spacing.ts

```diff
@@ -7,6 +7,11 @@ export const paddings: Rule[] = [
   [/^p-?([rltbse])-?(-?.+)$/, directionSize('padding'), { autocomplete: '(m|p)<directions>-<num>' }],
   [/^p-(block|inline)-(-?.+)$/, directionSize('padding'), { autocomplete: '(m|p)-(block|inline)-<num>' }],
   [/^p-?([bi][se])-?(-?.+)$/, directionSize('padding'), { autocomplete: '(m|p)-(bs|be|is|ie)-<num>' }],
+  [/^p-?([xy])()$/, directionSize("padding")],
+  [/^p-?([rltbse])()$/, directionSize("padding")],
+  [/^p-(block|inline)()$/, directionSize("padding")],
+  [/^p-?([bi][se])()$/, directionSize("padding")],
+  [/^p-?xy()()$/, directionSize("padding")]
 ]
```

Comment:
Please move the added rules next to the original to allow better override

ex:
```ts
  [/^p-?([xy])-?(-?.+)$/, directionSize('padding')],
  [/^p-?([xy])()$/, directionSize("padding")],
  [/^p-?xy()()$/, directionSize("padding")],
  // then
  [/^p-?([rltbse])-?(-?.+)$/, directionSize('padding'), { autocomplete: '(m|p)<directions>-<num>' }],
  /// ...
```

Also, please provide some test.